docs: DOC-193: Clarify legacy param for Taxonomy tag

caitlinwheeless opened the pull request:
Clarify that the legacy param preserves the ability to add labels on the fly. 



Affects:
- [X] Community docs
- [X] Enterprise docs

Commit:
docs: DOC-193: Clarify legacy param for Taxonomy tag

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8412171031

nikitabelonogov commented:
/git merge 
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8424504763)
> Successfully merged:  6 files changed, 484 insertions(+), 40 deletions(-)

nikitabelonogov committed:
Merge branch 'develop' into 'docs/193'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8424504763

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8424542325
